### ISSUE TITLE
NTR: Semen volume low in ejaculate  (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7955]
New term request Semen volume low in ejaculate  (elthat_100624040712): <2 mL  UMLS: 0855521

**Comment by MagoWyr** [comment]
Hi Peter,
in general, I'm fine with this term. However, "ejaculate" is commonly used as a synonym for "semen" so I suggest labelling the term "Low semen volume".
Best wishes,
Margot
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr  Thanks
@MickeySegal

adding as

Adding new term: 
- ID: HP:6000135
- label: Low semen volume
- parents: [Abnormal male reproductive system physiology (HP:0012874)](https://hpo.jax.org/app/browse/term/HP:0012874)
- pmid: 
- def: Volume of semen in ejaculate below the lower limit of normal. This finding can be ascertained by semen analysis. Comment: Sperm are produced within the testis, in the process of spermatogenesis, then travel through the male reproductive tract (the seminiferous tubules, epididymis, vas deferens, prostate and urethra). As the sperm travel, seminal fluid is added from the seminal vesicles, prostate, testis and epididymis and periurethral glands. The function of the ejaculate is to transport sperm into the female genital tract and to provide a suitable environment for sperm survival during this transit.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7955
- synonyms:
